src/processor_cxx11/main.cxx

```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+
+int main()
+{
+    std::cout << "I am processor with C++11..." << std::endl;
+    return 1;
```

Comment:
0 indicates no errors (at least it is a default linux behavior)

Reply:
That's why I returned 1 - it does nothing and shall be fixed.